usage/index.rst

```diff
@@ -0,0 +1,11 @@
+Usage
```

Comment:
I would have add this to the main readme as a new paragraph not as a separate file